profiles/package.mask

```diff
@@ -32,6 +32,13 @@
 
 #--- END OF EXAMPLES ---
 
+# Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com> (2020-04-16)
+# Very little activity for almost a year and the link to download the
+# "compiled" man-pages is dead.
+# Use app-i18n/man-pages-l10n[l10n_fr] as replacement.
+# Masked for removal in 30 days.
```

Comment:
Please add a bug number to the mask message. (No need to open a new bug, I think we can use [bug #717744](https://bugs.gentoo.org/717744) for this.)